ko-search/movies-evan-likes/src/App/components/Filters/index.js

```diff
@@ -1,8 +1,11 @@
-import React from 'react'
-import styles from './styles.css'
+import React, { Component } from 'react';
+import { centeredContainer } from 'sharedStyles.css';
+import { filter } from './styles.css';
 
-const Filters = () => (
-  <div className={styles.filters} />
-)
+const Filters = props =>
+  <div className={centeredContainer}>
+    <input className={filter} placeholder={'filter by title'} onChange={props.filterByTitle} />
+    <input className={filter} placeholder={'filter by decade'} onChange={props.filterByDecade} />
```

Comment:
I thought you may have meant that, but I wasn't 100% sure on the phrasing, so I just took a guess. Sorry!